ftva_lab_data/urls.py

```diff
@@ -22,4 +22,9 @@
         views.get_external_search_results,
         name="get_external_search_results",
     ),
+    path(
+        "metadata_json/<int:record_id>/<str:inventory_number>/",
```

Comment:
I think the URL should not include the inventory number. The view already has to retrieve the record itself, and can easily pull the inventory number out of that.  Removing it from the URL also simplifies what the calling template has to do.